ginivision/src/main/java/net/gini/android/vision/internal/qrcode/QRCodeDetectorTask.java

```diff
@@ -21,8 +21,15 @@
     List<String> detect(@NonNull final byte[] image, @NonNull final Size imageSize,
             final int rotation);
 
-    boolean isOperational();
+    void isOperational(@NonNull final Callback callback);
```

Comment:
IMO the name of this method is a little bit confusing now that is returning a void. Maybe `checkAvailability` or similar could be more precise.